okhttp/src/main/java/com/squareup/okhttp/internal/spdy/SpdyConnection.java

```diff
@@ -640,8 +640,11 @@ private Reader() {
             @Override public void execute() {
               try {
                 handler.receive(newStream);
-              } catch (IOException e) {
-                throw new RuntimeException(e);
+              } catch (RuntimeException | IOException e) {
```

Comment:
Sorry - you'll have to specify what it should do. This was just a guess to get the tests to pass.

Previously, if an exception was thrown it was being swallowed everywhere.
Now on desktop it's being logged and kills the thread.
On Android it's killing the process.